cmd/dep/ensure_test.go

```diff
@@ -186,3 +186,102 @@ func TestDeduceConstraint(t *testing.T) {
 		}
 	}
 }
+
+func TestEnsureUpdate(t *testing.T) {
+	test.NeedsExternalNetwork(t)
+	test.NeedsGit(t)
+
+	h := test.NewHelper(t)
+	defer h.Cleanup()
+
+	h.TempDir("src")
+	h.Setenv("GOPATH", h.Path("."))
+
+	m := `package main
+
+import (
```

Comment:
I just merged #215 (sorry), so please rebase this branch and switch these fixtures to using the golden file pattern.